### DISCUSSION TITLE
Battery health degraded error

### DISCUSSION THREAD

**Discussion by 27andrea** [teslamate-org/teslamate#3339] [Q&A]
Goodmorning,i have seen that my battery health degradation is 3,7%.i check and there are some charges that make this error.
How Can i solve this is sue? Thanks![image](https://github.com/adriankumpf/teslamate/assets/5239608/651a6477-6c7e-4c2f-9ac7-dac3ef718ad1)![image](https://github.com/adriankumpf/teslamate/assets/5239608/ae7ca9c5-4219-4c7c-a14c-1d88e0445ebe)

**Comment by sparvata** [comment]
Not sure which dashboard this data is coming from. Did you try installing the battery health dashboard from here and check if the error persists? https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/tree/main/dashboards
Not that I am a super expert and may know the problem, but just providing additional information.
